cachito/web/api_v1.py

```diff
@@ -61,6 +61,7 @@ def get_status_short():
         status(short=True)
         retval = {"ok": True}
     except CachitoError as e:
+        flask.current_app.logger.error(e)
```

Comment:
Is the goal to extract the /status/short error messages from logs? Would an easily recognizable prefix be helpful? Something like `status is not OK: %s`

Reply:
yes please, use a prefix

Reply:
Yes, that's the intention. I omitted the prefix since we already have the exact location the error happened in the log:

```
[2021-10-18 11:48:05 cachito.web.app ERROR api_v1.get_status_short] nexus unavailable: None
```

Reply:
I would keep as it is.

Reply:
ack, works for me